docs/grpc/python.md

```diff
@@ -11,31 +11,31 @@ file in Python before you can use it to communicate with lnd.
 
 1. Create a virtual environment for your project
     ```shell
-    ⛰  virtualenv lnd
+    virtualenv lnd
     ```
 2. Activate the virtual environment
     ```shell
-    ⛰  source lnd/bin/activate
+    source lnd/bin/activate
     ```
 3. Install dependencies (googleapis-common-protos is required due to the use of
   google/api/annotations.proto)
     ```shell
-    lnd ⛰  pip install grpcio grpcio-tools googleapis-common-protos
+    lnd pip install grpcio grpcio-tools googleapis-common-protos
     ```
 4. Clone the google api's repository (required due to the use of
   google/api/annotations.proto)
     ```shell
-    lnd ⛰  git clone https://github.com/googleapis/googleapis.git
+    lnd git clone https://github.com/googleapis/googleapis.git
     ```
 5. Copy the lnd lightning.proto file (you'll find this at
   [lnrpc/lightning.proto](https://github.com/lightningnetwork/lnd/blob/master/lnrpc/lightning.proto))
   or just download it
     ```shell
-    lnd ⛰  curl -o lightning.proto -s https://raw.githubusercontent.com/lightningnetwork/lnd/master/lnrpc/lightning.proto
+    lnd curl -o lightning.proto -s https://raw.githubusercontent.com/lightningnetwork/lnd/master/lnrpc/lightning.proto
```

Comment:
Those commands are now invalid because of the `lnd` prefix which should be removed now.

Reply:
Ahh good call. Did not find any other instances, pushed!